src/tim/vx/graph.cc

```diff
@@ -190,6 +190,20 @@ bool GraphImpl::Compile() {
 
 bool GraphImpl::CompileToBinary(void* buf, size_t* size) {
   bool status = true;
+
+  auto major = vsi_nn_GetVersionMajor();
+  auto minor = vsi_nn_GetVersionMinor();
+  auto patch = vsi_nn_GetVersionPatch();
+
+  vsi_nn_SetGraphVersion(graph_, major, minor, patch);
+
+  bool is_fast_mode = options_.isRelaxMode();
+  if (is_fast_mode) {
+    VSILOGW("Important notice: float model executed in bfloat16 "
+            "mode which will have better performance but lower precesion");
+  }
+  vsi_nn_SetGraphFastMode(graph_, is_fast_mode);
```

Comment:
create a private function such as ConfigGraph() to set those information ? then we don't have to repeat same code.